docs/presigned-put-upload-via-browser.md

```diff
@@ -86,10 +102,8 @@ using this URL we upload directly to `play.minio.io:9000`.
          $('#status').text(`Uploaded ${file.name}.`)
        }
      }
- }
-
+}
 </script>
 ```
 
```

Comment:
James, i like that we removed "Server Code" and "Client Code" headings in this PR. And we created the TOC links on the top for each recipe. I think this is valuable. However, in the actual recipe steps, i think we lost the ability to cut copy paste entire snippet. We could add code comments and keep it as a copyable snippet. One of the hard things is to know how to reconstruct the full code when trying/learning.

Reply:
@deekoder I have made this change in the branch. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-7/docs/presigned-put-upload-via-browser.md.